pandas/core/groupby/ops.py

```diff
@@ -408,6 +408,7 @@ def _masked_ea_wrap_cython_operation(
 
         # Copy to ensure input and result masks don't end up shared
         mask = values._mask.copy()
```

Comment:
For an aggregation op like this, I guess we can avoid the mask copy since the mask is not being modified inplace. (but not something needs to be done in this pr, just a note)

Reply:
Though I suppose there's the tradeoff that then we'd lose mask contiguity guarantee in the algo?

Reply:
haven't looked at the contiguity but that makes sense